go.mod

```diff
@@ -159,3 +160,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/gobuffalo/packr/v2 => github.com/gobuffalo/packr/v2 v2.3.2
```

Comment:
is this replace for debugging process or is it valid ?

Reply:
I think I added it to fix a security report, but this library is not a dependency anymore, since #1077. I will remove this line.